src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -493,6 +493,16 @@ public TransportNetworks(Context context) {
 						.setStatus(TransportNetwork.Status.APLHA)
 		);
 
+		// Canada
+		region = getString(R.string.np_region_canada);
+
+		list.add(new TransportNetwork(context, NetworkId.ONTARIO)
+				         .setName("Ontario")
+				         .setDescription("Ottawa, Toronto")
```

Comment:
I just noticed after merging that these are not translatable strings. Are you sure these names are the same in every language?